modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/filter/ElasticsearchFilterTranslator.java

```diff
@@ -48,7 +48,7 @@
  */
 @Component(
 	property = "search.engine.impl=Elasticsearch",
-	service = {FilterToQueryBuilderTranslator.class, FilterTranslator.class}
```

Comment:
@MrJohn1911 In the cases where such interface gets removed from `service` array, it could also be removed from `implements`, because such kind of interfaces is only their for direct reference.

At least for FilterToQueryBuilderTranslator and QueryToQueryBuilderTranslator, my IntelliJ shows only 1 usage which is the `implements`.

Please rescan the changes and remove these interfaces.

Reply:
@MrJohn1911 By the way, in this class, method `setQueryFilterTranslator` doesn't seem to be used by anyone. Could you double check, and remove it?